UnicornRender/video/source/Primitives.cpp

```diff
@@ -128,9 +147,143 @@ Mesh& Primitives::Sphere(Mesh& mesh, float radius, uint32_t rings, uint32_t sect
         }
     }
 
-    mesh.SetMeshData(vertices, indices);
+    mesh->SetMeshData(vertices, indices);
 
     return mesh;
 }
+
+namespace
+{
+
+Mesh* ProcessMesh(aiMesh* mesh, aiScene const* scene, std::string const& dir);
+
+void ProcessNode(aiNode* node, aiScene const* scene, std::string const& dir, std::list<Mesh*>& meshes)
```

Comment:
You should document functions even if those are anonymous and are only exist in one translation unit. This function name is very generic and does not describe what this function does.

Reply:
Same goes for all the functions below in this file.